### ISSUE TITLE
manpages broken

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#5419]
```
$ man zstd
 man: command exited with status 255: (cd /usr/local/share/man && /usr/local/libexec/man-db/zsoelim) | (cd /usr/local/share/man && /usr/local/libexec/man-db/manconv -f UTF-8:ISO-8859-1 -t ANSI_X3.4-1968//IGNORE) | (cd /usr/local/share/man && tbl) | (cd /usr/local/share/man && nroff -mandoc -rLL=145n -rLT=145n -Tascii)
```

**Comment by uberhacker** [comment]
A fresh install did not fix the problem.

**Comment by satmandu** [comment]
Does `PAGER=most man zstd` work?

**Comment by satmandu** [comment]
Anything useful from ```ldd `which man` ```?

**Comment by uberhacker** [comment]
My pager is already set to most.

**Comment by satmandu** [comment]
Without PAGER set this is what I see:
```
man zstd
man: can't execute /usr/bin/less: No such file or directory
man: command exited with status 255: (cd /usr/local/share/man && /usr/local/libexec/man-db/zsoelim) | (cd /usr/local/share/man && /usr/local/libexec/man-db/manconv -f UTF-8:ISO-8859-1 -t ISO-8859-1//IGNORE) | (cd /usr/local/share/man && tbl) | (cd /usr/local/share/man && nroff -mandoc -rLL=96n -rLT=96n -Tutf8)
```

**Comment by uberhacker** [comment]
```
$ lld `which man`
lld is a generic driver.
Invoke ld.lld (Unix), ld64.lld (macOS), lld-link (Windows), wasm-ld (WebAssembly) instead
```

**Comment by satmandu** [comment]
`ldd` not `lld`

**Comment by uberhacker** [comment]
Does mandb compile for you?

**Comment by satmandu** [comment]
It did once... let me try again.

**Comment by satmandu** [comment]
FYI, my MO today has been to remove the HOST and TARGET entries from CREW_OPTIONS....

**Comment by satmandu** [comment]
hmmm.. Seeing same error as you though.

**Comment by satmandu** [comment]
The problem is a lack of `groff`, which needs to be listed as a dep of `mandb`.

install `groff` and the problem goes away.

**Comment by uberhacker** [comment]
Yep, I figured that out.  This is why we need to be careful we don't remove core packages from a core package. :)
